core/runtime/src/main/java/io/quarkiverse/operatorsdk/runtime/CRDConfiguration.java

```diff
@@ -31,6 +31,7 @@
 
     /**
      * Whether the extension should automatically apply updated CRDs when they change.
+     * When running on DEV mode, the CRD changes will always be applied automatically.
```

Comment:
This isn't true, though, because the default value for `apply` is still `false`. In fact, perhaps we should remove this property altogether and make it an automatic behavior during dev mode?

Reply:
I removed this code: https://github.com/quarkiverse/quarkus-operator-sdk/pull/260/files#diff-e6fe47bd3b62e0de50164805a7f94703596cb45bd7deae0e5c9f0f554930773cL54
Therefore, this property is no longer used when running in DEV mode and hence the CRDs will always be applied.
I double-checked that this is now working like this.

Reply:
Indeed, I missed it looking at the PR from the web interface's diffs.